config/test.exs

```diff
@@ -1,5 +1,5 @@
 use Mix.Config
 
 config :spreedly,
-  environment_key: "R7lHscqcYkZeDGGbthKp6GKMu15",
-  access_secret: "8sefxO5Q44sLWpmZpalQS3Qlqo03JbCemsqsWJR3YOLCuigOFRlaLSAn0WaL5dWU"
+  environment_key: System.get_env("SPREEDLY_ENVIRONMENT_KEY"),
+  access_secret: System.get_env("SPREEDLY_ACCESS_SECRET")
```

Comment:
I'm also not against importing a "test.secret.exs" that's ignore by git.

Reply:
I do dig having a gitignored file since then I can set it up once in a way that's local to the project and easily run tests in the future. Is there a way to use the env var approach that doesn't feel "global"?

Reply:
Since these are `test` only credentials, I'd recommend using the `test.secret.exs` file approach with updates to the Readme to indicate that a `test.secret.exs` with valid credentials is required to run tests.

Reply:
With gitignored a file, config/test.exs becomes:

```
use Mix.Config

import_config "test.secret.exs"
```

Then in config/test.secret.exs:

```
use Mix.Config

config :spreedly,
  environment_key: System.get_env("SPREEDLY_ENVIRONMENT_KEY"),
  access_secret: System.get_env("SPREEDLY_ACCESS_SECRET")
```

The test.secret.exs doesn't have to use ENVs though, the credentials could be hardcoded in since it's local to the project and ignored.

Reply:
I'd skip the ENV's in the Readme example `test.secret.exs` and hard code them. ENV's are extra work at that point.

Reply:
Yeah, going with the import_config approach, I'm not going to mention ENVs.

Reply:
PR updated. Take another look please.

Reply:
@dtykocki What do you think about requiring the the `secret` file in `test.exs`, like we do in Dashboard https://github.com/spreedly/calamari/blob/master/apps/dashboard/config/test.exs#L25

Reply:
I like. Adding.

Reply:
Added.